docs/run-a-node/setup/install.md

```diff
@@ -289,9 +300,18 @@ Genesis hash: mFgazF+2uRS1tMiL9dsj01hJGySEmPN28B/TjjvpVW0=
 
 `./goal node catchup 4420000#Q7T2RRTDIRTYESIXKAAFJYFQWG4A3WRA3JIUZVCJ3F4AQ2G2HZRA -d ~/node/datafastcatchup`
 
+   Alternate syntax
+ 
+ `./goal node catchup 7560000#4420000#Q7T2RRTDIRTYESIXKAAFJYFQWG4A3WRA3JIUZVCJ3F4AQ2G2HZRA -d datafastcatchup`
```

Comment:
The `7560000#4420000#Q7T2RRTDIRTYESIXKAAFJYFQWG4A3WRA3JIUZVCJ3F4AQ2G2HZRA` is not a valid catchpoint label. It includes two different rounds. For the purpose of this manual, I suggest we'll stick to a single catchpoint label.